src/lib/ng-package/discover-packages.ts

```diff
@@ -124,20 +131,20 @@ async function findSecondaryPackagesPaths(directoryPath: string, excludeFolder:
 /**
  * Reads a secondary entry point from it's package file.
  *
- * @param primaryDirectoryPath A path pointing to the directory of the primary entry point.
  * @param primary The primary entry point.
+ * @param userPackage The user package for the secondary entry point.
  */
 function secondaryEntryPoint(
-  primaryDirectoryPath: string,
   primary: NgEntryPoint,
-  { packageJson, ngPackageJson, basePath }: UserPackage,
+  userPackage: UserPackage,
 ): NgEntryPoint {
-  if (path.resolve(basePath) === path.resolve(primaryDirectoryPath)) {
+  const { packageJson, ngPackageJson, basePath } = userPackage;
```

Comment:
I've moved this assignment from the param list in order improve autocomplete suggestions.